ui/components/component-library/text/text.scss

```diff
@@ -78,7 +78,7 @@ $text-variants: (
 
   @each $overflow in $overflow-wrap {
     &--overflow-wrap-#{$overflow} {
-      overflow-wrap: $overflow;
+      word-break: $overflow;
```

Comment:
The property values of `$overflow-wrap` don't all work with word-break? e.g. `anywhere`
https://github.com/MetaMask/metamask-extension/blob/f82514736adae8eb4048b5ab9e7c7b64b15e0bbe/ui/css/design-system/attributes.scss#L47

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Do we need to create a wordbreak scss variable? cc @garrettbear

Reply:
Sweet created an issue. Can you fill in the deets? https://github.com/MetaMask/metamask-extension/issues/18354